### PR TITLE
Adding one more card to chaosbot.com

### PR DESCRIPTION
**Pull request by ipranjal**
This card can be used to display interesting concepts laid down for chaos bot related to disscussion on  #479 and tangentially related to #473

### CODE REVIEW

File: server/static/js/chaos.js [hunk 1]
```diff
@@ -126,14 +126,27 @@ document.getElementById("voters").onclick = () => {
...
+    document.getElementById("main").classList.remove("active");
+    document.getElementById("future").classList.remove("inactive");
+};
+
 document.getElementById("main").onclick = () => {
     if (document.getElementById("main").classList.contains("inactive")) {
         document.getElementById("main").classList.add("active");
         document.getElementById("voters").classList.add("inactive");
         document.getElementById("voters").classList.remove("active");
+         document.getElementById("future").classList.add("inactive");
```
Comment (andrewda) [inline server/static/js/chaos.js]: fix spacing

**Review by andrewda — CHANGES_REQUESTED** [review]

File: server/static/js/chaos.js (hunk 1 above)
Comment (ipranjal) [reply server/static/js/chaos.js]: yep was just doing that

**Review by andrewda — APPROVED** [review]

**Comment by ipranjal** [comment]
@PlasmaPower you dont seem much happy about this :P

**Comment by PlasmaPower** [comment]
@physcocode no, the point of a wiki is sorta that it's easier to change (you don't need a PR). This defeats the point.

**Comment by andrewda** [comment]
It'd be cool if this could either a) just link to the feature request wiki page, or b) scrape the wiki page and display the content.

**Comment by ipranjal** [comment]
@PlasmaPower @andrewda just wanna give democracy over what people actually wanted to display on the list of future concept . people could down vote what they don't want to add in list

**Comment by mdcfe** [comment]
👎 for two reasons:
* Static content on the homepage - as @PlasmaPower said, we can use the wiki
* The cards on the homepage are already clunky and don't work well on mobile. If we can improve how they work and look (for example no animations on mobile, maximum height to prevent scrollbars) then I'd be fine with adding more cards.

**Comment by ipranjal** [comment]
Okay I am down voting this the card also don't work ok safari it's frikin broken

**Pull request closed without merging**